e2e/playwright/boolean.spec.ts

```diff
@@ -114,6 +114,22 @@ test.describe('Point and click for boolean workflows', () => {
 
         await editor.expectEditor.toContain(operation.code)
       })
+
+      await test.step(`Delete ${operationName} operation via feature tree selection`, async () => {
+        await toolbar.openPane('feature-tree')
+        const ftOp = await toolbar.getFeatureTreeOperation(operationName, 0)
+        await ftOp.click({ button: 'left' })
+        await page.keyboard.press('Delete')
+        await scene.settled(cmdBar)
+        await toolbar.closePane('feature-tree')
+
+        // Expect changes in ft and code
+        await toolbar.openPane('code')
+        await editor.expectEditor.not.toContain(operation.code)
+        await expect(
+          await toolbar.getFeatureTreeOperation(operationName, 0)
+        ).not.toBeVisible()
+      })
```

Comment:
Somehow this wasn't working well locally

Reply:
Which line was failing? I'm guessing this one?

```ts
        await editor.expectEditor.not.toContain(operation.code)
```

Reply:
It just doesn't delete in the test, so weird. Working on it now

Reply:
Went with right click -> Delete and that seems much better

Reply:
Or is it possible that execution and population of the feature tree were taking longer than the test could allow?

Reply:
I also added scene.settled, should be better I think!